From 0f06c4cb4a57083c08312144b03c8346cf620be6 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 30 Oct 2020 22:15:29 +0000
Subject: [PATCH] json-glib/json-enum-types.c.in: fix build reproducibility

Changes the comment and include to reference the file basename
instead of the full path. This ensures that the generated file is
reproducible when it is included in source packages meant for debugging.

Upstream-Status: Backport [6f3842abd5dff68b6ee5f2ef48a4ebf1cbebf434]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 json-glib/json-enum-types.c.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/json-glib/json-enum-types.c.in b/json-glib/json-enum-types.c.in
index da07c46..7b39290 100644
--- a/json-glib/json-enum-types.c.in
+++ b/json-glib/json-enum-types.c.in
@@ -9,8 +9,8 @@
 
 /*** BEGIN file-production ***/
 
-/* enumerations from "@filename@" */
-#include "@filename@"
+/* enumerations from "@basename@" */
+#include "@basename@"
 
 /*** END file-production ***/
 
